SU2_CFD/include/fluid/CPengRobinson.hpp

```diff
@@ -110,7 +110,7 @@ class CPengRobinson final : public CIdealGas {
    */
   void ComputeDerivativeNRBC_Prho(su2double P, su2double rho) override;
```

Comment:
Where applicable switch the old overrides to final (now that this class is not final anymore).
```suggestion
  void ComputeDerivativeNRBC_Prho(su2double P, su2double rho) final;
```